content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -27,7 +27,7 @@ need the features they provide.
 
 Namespaces provide a scope for names.  Names of resources need to be unique within a namespace,
 but not across namespaces. Namespaces can not be nested inside one another and each Kubernetes 
-resources can only be in one namespace.
+resources can only be one per namespace.
```

Comment:
The current grammar isn't quite right, is it?

How about:
```suggestion
resource can only be in one namespace.
```

That makes sense to me.